src/acl.c

```diff
@@ -2655,6 +2659,7 @@ void addACLLogEntry(client *c, int reason, int context, int argpos, sds username
 
     le->cinfo = catClientInfoString(sdsempty(), realclient, 0);
     le->context = context;
+    le->scripting_engine_name = scripting_engine_name ? sdsdup(scripting_engine_name) : sdsempty();
```

Comment:
`scripting_engine_name` appears to be coming from globals, so do we really need to pass it as a parameter to `addACLLogEntry()`? Can't we just fetch it here, like this:

```suggestion
    le->scripting_engine_name = scriptIsRunning() ? scriptGetRunningEngineName() : sdsempty();
```